evaluation/mint/env.py

```diff
@@ -0,0 +1,119 @@
+import re
+import traceback
+from typing import Dict, Optional
+
+from datatypes import ParseError, StepOutput, TaskState
+from task import Task
+
+from opendevin.controller.state.state import State
+
+
+class SimplifiedEnv:
+    INVALID_INPUT_MESSAGE = (
+        "I don't understand your input. \n"
+        'If you want to execute code, please use <execute> YOUR_CODE_HERE </execute>.\n'
```

Comment:
I think you can replace these "`<execute>`" tag with "`<execute_ipython>`" tag used by CodeActAgent.

Then you can add an assertion for `Agent`: The agent needs to be `CodeActAgent` to run MINT. We can add supports for other agents when needed.

Reply:
Nice idea. I've changed it! Also for `CodeActAgent`, I hardcoded it for the `AGENT` option inside `run_infer.sh`.